### PR TITLE
Add instructions on how to request an RRA

### PR DESCRIPTION
**Pull request by tristanweir**
Making this a PR since I'm not sure if the intent of docs in this space is to be fully Mozilla-agnostic. But if we refer people to this space, they should know how to kick off an RRA.

### CODE REVIEW

**Comment by gdestuynder** [comment]
I would say it's a balance. The more "generic" it is the better, because it forces us to have processes, docs, etc that will work for all Mozilla use cases - and can also be re-used externally.

That said, some Mozilla-specific things can make sense and I think a link does. Hopefully, we don't get requests from non-mozilla too often though, as we would be unable to serve them (though, it'd be interesting to know if third parties would like this to be a thing)

File: docs/guidelines/risk/rapid_risk_assessment.md [hunk 1]
```diff
@@ -62,6 +62,12 @@ Firefox accounts store user data:
 
 The RRA risk table facilitates discovering the answers to these questions.
 
+# How-to: Request an RRA be run on your service
```
Comment (gdestuynder) [inline docs/guidelines/risk/rapid_risk_assessment.md]:
```suggestion
# How-to: Request an RRA be run for you service
```

File: docs/guidelines/risk/rapid_risk_assessment.md [hunk 2]
```diff
@@ -62,6 +62,12 @@ Firefox accounts store user data:
...
+
+(File a bug)[https://bugzilla.mozilla.org/enter_bug.cgi?product=Enterprise%20Information%20Security&component=Rapid%20Risk%20Analysis] to start an RRA. Include basic information about the project, including any relevant links and 0-2 additional people to include during the meeting. Our team will automatically triage the RRA and someone from InfoSec will reach out to you.
+
+Note that any projects started through CASA will be automatically evaluated, and bugs will be automatically filed to start the RRA.
```
Comment (gdestuynder) [inline docs/guidelines/risk/rapid_risk_assessment.md]:
```suggestion

```

File: docs/guidelines/risk/rapid_risk_assessment.md [hunk 3]
```diff
@@ -62,6 +62,12 @@ Firefox accounts store user data:
 
 The RRA risk table facilitates discovering the answers to these questions.
 
+# How-to: Request an RRA be run on your service
+
+(File a bug)[https://bugzilla.mozilla.org/enter_bug.cgi?product=Enterprise%20Information%20Security&component=Rapid%20Risk%20Analysis] to start an RRA. Include basic information about the project, including any relevant links and 0-2 additional people to include during the meeting. Our team will automatically triage the RRA and someone from InfoSec will reach out to you.
```
Comment (gdestuynder) [inline docs/guidelines/risk/rapid_risk_assessment.md]:
```suggestion
To manually request an RRA, please (file a bug)[https://bugzilla.mozilla.org/enter_bug.cgi?product=Enterprise%20Information%20Security&component=Rapid%20Risk%20Analysis] to our component. Please include basic information about the project, a diagram, any relevant links and 0-2 additional people to invite during for the assessment.
```

File: docs/guidelines/risk/rapid_risk_assessment.md (hunk 1 above)
Comment (floatingatoll) [reply docs/guidelines/risk/rapid_risk_assessment.md]: I feel "Request an RRA of your service" makes more sense here when considering the expanded acronym — "Request a Rapid Risk Analysis of your service".

**Pull request merged**